usr/local/www/services_captiveportal_filemanager.php

```diff
@@ -135,100 +137,162 @@ function cpelements_sort() {
-		<a href="services_captiveportal_filemanager.php?zone=<?=$cpzone;?>"><img src="/themes/<?php echo $g['theme']; ?>/images/icons/icon_x.gif" title="<?=gettext("cancel"); ?>" width="17" height="17" border="0" alt="delete" /></a>
-		</td>
-	  </tr>
-  <?php else: ?>
-	  <tr>
-		<td class="list" colspan="2"></td>
-		<td class="list">
-			<table border="0" cellspacing="0" cellpadding="1" summary="add">
-			    <tr>
-				<td width="17" height="17"></td>
-				<td><a href="services_captiveportal_filemanager.php?zone=<?=$cpzone;?>&amp;act=add"><img src="/themes/<?php echo $g['theme']; ?>/images/icons/icon_plus.gif" title="<?=gettext("add file"); ?>" width="17" height="17" border="0" alt="add" /></a></td>
-			    </tr>
-			</table>
-		</td>
-	  </tr>
-  <?php endif; ?>
-	</table>
-	<span class="vexpl"><span class="red"><strong>
-	<?=gettext("Note:"); ?><br />
-	</strong></span>
+// The notes displayed on the page are large, the page content comparitively small. A "Note" button
+// is provided so that you only see hte notes if you ask for them
+?>
+<div class="notes panel panel-default">
+	<div class="panel-heading">Notes</div>
+	<div class="panel-body">
 	<?=gettext("Any files that you upload here with the filename prefix of captiveportal- will " .
 	"be made available in the root directory of the captive portal HTTP(S) server. " .
 	"You may reference them directly from your portal page HTML code using relative paths. " .
 	"Example: you've uploaded an image with the name 'captiveportal-test.jpg' using the " .
-	"file manager. Then you can include it in your portal page like this:"); ?><br /><br />
-	<tt>&lt;img src=&quot;captiveportal-test.jpg&quot; width=... height=...&gt;</tt>
-	<br /><br />
-	<?=gettext("In addition, you can also upload .php files for execution.  You can pass the filename " .
-	"to your custom page from the initial page by using text similar to:"); ?>
-	<br /><br />
-	<tt>&lt;a href="/captiveportal-aup.php?zone=$PORTAL_ZONE$&amp;redirurl=$PORTAL_REDIRURL$"&gt;<?=gettext("Acceptable usage policy"); ?>&lt;/a&gt;</tt>
-	<br /><br />
-	<?php printf(gettext("The total size limit for all files is %s."), format_bytes($g['captiveportal_element_sizelimit']));?></span>
-</td>
-</tr>
-</table>
-</form>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+	"file manager. Then you can include it in your portal page like this:")?><br /><br />
+	<tt>&lt;img src=&quot;captiveportal-test.jpg&quot; width=... height=...&gt;</tt><br /><br />
+	<?=gettext("In addition, you can also upload .php files for execution.	You can pass the filename " .
+	"to your custom page from the initial page by using text similar to:")?><br /><br />
+	<tt>&lt;a href="/captiveportal-aup.php?zone=$PORTAL_ZONE$&amp;redirurl=$PORTAL_REDIRURL$"&gt;<?=gettext("Acceptable usage policy"); ?>&lt;/a&gt;</tt><br /><br />
+	<?=sprintf(gettext("The total size limit for all files is %s."), format_bytes($g['captiveportal_element_sizelimit']))?>
+	</div>
+</div>
+
+<script>
```

Comment:
This script is correctly written, but why would this page need toggleable notes? I'd suggest using class="help-block" instead; then we could toggle all helptexts globally, at once